# human pose
geometry_msgs/Point nose
geometry_msgs/Point left_eye
geometry_msgs/Point right_eye
geometry_msgs/Point left_ear
geometry_msgs/Point right_ear
geometry_msgs/Point left_shoulder
geometry_msgs/Point right_shoulder
geometry_msgs/Point left_elbow
geometry_msgs/Point right_elbow
geometry_msgs/Point left_wrist
geometry_msgs/Point right_wrist
geometry_msgs/Point left_hip
geometry_msgs/Point right_hip
geometry_msgs/Point left_knee
geometry_msgs/Point right_knee
geometry_msgs/Point left_foot
geometry_msgs/Point right_foot

# frame
robot_arm_follow_demo/Frame frame
